packages/server/src/profile/user-course.entity.ts

```diff
@@ -32,4 +34,8 @@ export class UserCourseModel extends BaseEntity {
 
   @Column({ type: 'enum', enum: Role, default: Role.STUDENT })
   role: Role;
+
+  @Exclude()
+  @OneToMany((type) => EventModel, (event) => event.user)
+  events: EventModel[];
```

Comment:
Does this work given that the event model does not have a `UserCourse` foreign key?

Reply:
ah shoot I fogot to remove this on this side -- i'll get it when I get home